k8s/src/test/scala/io/buoyant/k8s/IngressCacheTest.scala

```diff
@@ -1,12 +1,228 @@
 package io.buoyant.k8s
 
+import com.twitter.finagle.Service
+import com.twitter.io.Buf
+import com.twitter.util.Future
 import io.buoyant.test.Awaits
 import org.scalatest.FunSuite
+import com.twitter.finagle.http.{Request => FRequest, Response => FResponse}
```

Comment:
TIOLI, but as far as I can tell, the renaming import shouldn't be necessary here?